src/components/z-breadcrumb/index.tsx

```diff
@@ -0,0 +1,437 @@
+import {Component, Prop, h, State, Host, Listen, Element, Event, Watch} from "@stencil/core";
+import {
+  BreadcrumbHomepageVariant,
+  BreadcrumbPath,
+  BreadcrumbPathStyle,
+  Device,
+  KeyboardCode,
+  ListDividerType,
+  ListSize,
+  PopoverPosition,
+} from "../../beans";
+import {EventEmitter} from "puppeteer";
+import {getDevice, handleKeyboardSubmit} from "../../utils/utils";
+
+@Component({
+  tag: "z-breadcrumb",
+  styleUrl: "styles.css",
+  shadow: true,
+})
+export class ZBreadcrumb {
+  /* Accessibility references */
+  /* Overflow-menu: https://www.w3.org/WAI/ARIA/apg/patterns/menu-button/ */
+  /* Breadcrumb: https://www.w3.org/WAI/ARIA/apg/patterns/breadcrumb/ */
+  /* Focus on multiline link: https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance.html */
+
+  @Element() hostElement: HTMLZBreadcrumbElement;
+
+  /** [optional] Path elements */
+  @Prop()
+  paths?: BreadcrumbPath[] | string;
```

Comment:
So che rompo le scatole, ma provando ad usare il componente, come se fossi un dev utente del ds, ho dovuto fare il reverse engineering del link home, per capire se Home fosse da scrivere nel text o se fosse libero. Allora mi è venuto un dubbio, ma valutate voi, con pazienza: non è che è meglio escludere Home dai path e metterlo in una proprietà apposita, non so, "homeLink"?

Reply:
Abbiamo scelto di includere il link home nell'array dei percorsi in modo da avere una gestione omogenea in entrambi i casi (path passati tramite prop oppure tramite children). Nel secondo caso infatti saremmo comunque "costretti" a passare il link home tramite children in modo che possa avere la sovrastruttura di cui eventualmente necessita il prodotto (es. Link di next che wrappa il link vero e proprio).

Aggiorniamo la documentazione per rendere più chiara la cosa.